docs/releases/upgrading/upgrade-v10.0.md

```diff
@@ -0,0 +1,110 @@
+# Upgrading from v9.0 to v10.0
+
+## Prerequisites
+
+The steps listed in this article require an existing local installation of InvenioRDM v9.0, please make sure that this is given!
+
+If unsure, run `invenio-cli install` from inside the instance directory before executing the listed steps.
+
+!!! warning "Backup"
+
+    Always backup your database and files before you try to perform an upgrade.
+
+!!! info "Older versions"
+
+    In case you have an InvenioRDM installation older than v9.0, you can gradually upgrade using the existing instructions to v9.0 and afterwards continue from here.
+
+## Upgrade Steps
+
+!!! warning "Upgrade your invenio-cli"
+
+    Make sure you have the latest `invenio-cli`, for InvenioRDM v10 the release is v1.0.5
+
+### Installing the Latest Versions
+
+Bump the RDM version, create a folder for your custom fields and rebuild the assets:
+
+```bash
+# Upgrade to InvenioRDM v10
+invenio-cli packages update 10.0.0
```

Comment:
Is this correct? I see [here](https://inveniordm.docs.cern.ch/reference/cli/#packages-update) the command is not yet supported or it misses documentation!

Reply:
They are supported, bad docs <https://github.com/inveniosoftware/invenio-cli/blob/master/invenio_cli/cli/packages.py#L79>